rllib/tests/test_supported_multi_agent.py

```diff
@@ -56,25 +54,41 @@ def tearDownClass(cls) -> None:
 
     def test_a3c_multiagent(self):
         check_support_multiagent(
-            "A3C", {"num_workers": 1, "optimizer": {"grads_per_step": 1}}
+            "A3C",
+            (
+                get_trainable_cls("A3C")
```

Comment:
soft NIT: Can we instead replace calls to get_trainable_cls.(ALGO).get_default_config with the actual config object?

Reply:
Yup, that'd be much better.

Reply:
will fix

Reply:
done